src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -424,3753 +416,3744 @@ private void BuildApplicationInitializerBody(IndentedStringBuilder writer, XamlO
+				.ToArray();
+
+			if (resourcesTogenerateApplyCompiledBindings.Any())
+			{
+				using (writer.BlockInvariant("Loading += (s, e) =>"))
+				{
+					foreach (var namedResource in resourcesTogenerateApplyCompiledBindings)
+					{
+						var type = GetType(namedResource.Value.Type);
+
+						writer.AppendLineInvariant($"{namedResource.Key}.ApplyCompiledBindings();");
+					}
+				}
+				writer.AppendLineInvariant(0, ";");
+			}
+		}
+
+		private void BuildInlineStyle(IIndentedStringBuilder writer, XamlObjectDefinition style)
+		{
+			TryAnnotateWithGeneratorSource(writer);
+			var targetTypeNode = GetMember(style, "TargetType");
+			var targetType = targetTypeNode.Value.ToString();
+			var fullTargetType = FindType(targetType).SelectOrDefault(t => t.ToDisplayString(), targetType);
+
+			using (writer.BlockInvariant("new global::Windows.UI.Xaml.Style(typeof({0}))", GetGlobalizedTypeName(fullTargetType)))
+			{
+				var basedOnNode = style.Members.FirstOrDefault(o => o.Member.Name == "BasedOn");
+
+				if (basedOnNode != null)
+				{
+					writer.AppendLineInvariant("BasedOn = (global::Windows.UI.Xaml.Style){0},", BuildBindingOption(basedOnNode, _styleSymbol, prependCastToType: false));
+				}
+
+				using (writer.BlockInvariant("Setters = ", fullTargetType))
+				{
+					var contentNode = FindMember(style, "_UnknownContent");
+
+					var settersNode = FindMember(style, "Setters");
+
+					if (contentNode != null && settersNode != null)
+					{
+						throw new InvalidOperationException($"The property 'Setters' is set more than once at line {style.LineNumber}:{style.LinePosition} ({_fileDefinition.FilePath}).");
+					}
+
+					contentNode = contentNode ?? settersNode;
+
+					if (contentNode != null)
+					{
+						foreach (var child in contentNode.Objects)
+						{
+							if (child.Type.Name == "Setter")
+							{
+								var propertyNode = FindMember(child, "Property");
+								var valueNode = FindMember(child, "Value");
+								var property = propertyNode?.Value.SelectOrDefault(p => p.ToString());
+
+								if (valueNode != null)
+								{
+									BuildPropertySetter(writer, fullTargetType, ",", property, valueNode);
+								}
+								else
+								{
+									GenerateError(writer, $"No value was specified for {property} on {child.Type.Name}");
+								}
+							}
+							else
+							{
+								GenerateError(writer, "Support for {0} is not implemented", child.Type.Name);
+							}
+						}
+					}
+				}
+			}
+		}
+
+		private void BuildPropertySetter(IIndentedStringBuilder writer, string fullTargetType, string lineEnding, string property, XamlMemberDefinition valueNode, string targetInstance = null)
+		{
+			TryAnnotateWithGeneratorSource(writer);
+			targetInstance = targetInstance ?? "o";
+
+			property = property?.Trim('(', ')');
+
+			// Handle attached properties
+			var isAttachedProperty = property?.Contains(".") ?? false;
+			if (isAttachedProperty)
+			{
+				var separatorIndex = property.IndexOf(".");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this call to 'property.IndexOf' to an overload that accepts a 'StringComparison' as a parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5305744)